static/js/flow/dashboardTypes.js

```diff
@@ -3,11 +3,15 @@ import Decimal from 'decimal.js-light';
 import type { Program } from './programTypes';
 
 // likely to change in very near future
-export type Dashboard = Array<Program>;
+export type Dashboard = {
+  programs:   Array<Program>,
+  is_edx_data_fresh: boolean
```

Comment:
Oh yes, you're right - I forgot that we had changed this around relatively recently. Carry on!